ufo/static/addUserModal.html

```diff
@@ -0,0 +1,132 @@
+<link rel="import" href="bower_components/paper-tabs/paper-tabs.html" />
+
+<dom-module id="add-user-modal">
+  <style is="custom-style">
+    paper-tabs {
+      --paper-tabs-selection-bar-color: #25CFB9;
+      background-color: white;
+    }
+    paper-tab {
+      --paper-tab-ink: #25CFB9;
+      color: black;
+      background-color: white;
+    }
+    h2.title {
+      background-color: white;
+    }
+    #ironPagesHolder {
+      background: #EEE;
+      margin-bottom: 0px;
+      border-bottom: 24px solid #EEE;
+    }
+    #spinner {
+      position: fixed;
+      top: 50%;
+      left: 50%;
+      z-index: 100000000;
+    }
+  </style>
+  <template>
+    <paper-dialog id="{{resources.modalId}}" modal>
+      <h2 class="title">{{resources.addText}}</h2>
+
+      <!--Portion for finding users via group, individual, domain, or manual
+      specification. -->
+      <paper-tabs id="tabs" selected="{{ selected }}">
+        <template is="dom-repeat" items="[[resources.addFlowTextDicts]]">
+          <paper-tab>{{item.tab}}</paper-tab>
+        </template>
+      </paper-tabs>
+
+      <div id="ironPagesHolder">
+        <iron-pages id="pages" selected="{{ selected }}">
+          <template is="dom-repeat" items="[[resources.addFlowTextDicts]]">
+            <template is="dom-if" if="{{item.isManual}}">
+              <div>
+                <template is="dom-if" if="{{isSpinnerOn}}">
+                  <paper-spinner id="spinner" active$={{isSpinnerOn}}></paper-spinner>
+                </template>
+                <form is="iron-form" id="{{item.id}}" method="post" action="{{resources.addUrl}}" on-iron-form-presubmit="enableSpinner" on-iron-form-response="parsePostResponse">
+                  <paper-input label="{{item.label1}}" type="textbox" name="name" value="" id="manualUserName" required>
+                  </paper-input>
+                  <paper-input label="{{item.label2}}" type="email" name="email" value="" id="manualUserEmail" required
+                  auto-validate pattern="{{EMAIL_VALIDATION_PATTERN}}"
+                  error-message="{{EMAIL_VALIDATION_ERROR}}">
+                  </paper-input>
+                  <input type="hidden" name="users" id="manualUserInput" value="">
+                  <input type="hidden" name="_xsrf_token" value="{{getXsrfToken()}}">
+                  <br>
+                </form>
+                <div class="buttons">
+                  <paper-button class="anchor-button" dialog-dismiss>
+                  {{resources.dismissText}}
+                  </paper-button>
+                  <paper-button class="anchor-button" autofocus on-tap="submitPostForm">
+                  {{item.saveButton}}
+                  </paper-button>
+                </div>
+              </div>
+            </template>
+            <template is="dom-if" if="{{!item.isManual}}">
+              <user-add-form resources="{{resources}}" add-flow-text-obj="{{item}}"></user-add-form>
```

Comment:
It looks like this form can be re-used in the setup page to add users as well.  Do you expect this custom element to be just dropped in there (i.e. work outside of the modal window)?  It looks okay to me, but want to verify this.